src/client/packaging/pypi/CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 All notable future changes to the `delphi_epidata` python client will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+## [4.2.0] - 2023-09-27
+
+### Changed
+ 
+- Modify the signatures for several methods: endpoint no longer needs to be specified in the params dict under the "source" key, but becomes a mandatory parameter for the method:
+  - `async_epidata(param_list, batch_size=50)` → `async_epidata(endpoint, param_list, batch_size=50)`
```

Comment:
This doesn't seem to match the function definition. Does this need to be updated or does the function need updating?

Reply:
i have some text for the changelog that im about to push that addresses this